less/footer.less

```diff
@@ -1 +1,18 @@
-// Footer Styles
\ No newline at end of file
+// Footer Styles
+footer {
+    // width: 100%;
+    display: flex;
+   background-color: @footer-background-color;
+    .center2;
+    @media @phone{
```

Comment:
Great use of the variable `@phone` here! Should the breakpoint ever change, you can change it in one place, in one file!